AwsCryptographicMaterialProviders/runtimes/go/ImplementationFromDafny-go/StormTrackingCMC/externs.go

```diff
@@ -0,0 +1,19 @@
+package StormTrackingCMC
```

Comment:
TODO: Skeleton Implementaion / not tested. Blocked on Dafny Bug: https://t.corp.amazon.com/P157792076